lib/snapshot/runner.rb

```diff
@@ -226,7 +226,7 @@ def verify_helper_is_current
       helper_files.each do |path|
         content = File.read(path)
 
-        if content.include?("start.pressForDuration(0, thenDragToCoordinate: finish)")
+        unless content.include?("SnapshotHelperVersion [1.1]")
```

Comment:
We still need to support the other migration, so this should actually be multiple `if`s in this method